### DISCUSSION TITLE
HTML panel tabset and callouts parsing

### DISCUSSION THREAD

**Discussion by DaedalusInMaze** [quarto-dev/quarto-cli#10373] [Q&A] [answered]
Hi Quarto community, 

### Problem Description
[Env: Python 3.11, Quarto 1.5.55, Windows 11]
I have been using Quarto version 1.3.433 previously and just upgraded to 1.5.55. My file contains a callout within a panel tabset which is not generating the expected behaviour. 

In 1.3.433 it behaves like the first picture where a callout does not affect the panel tabset structure. After updating to 1.5.55, it performs as the second picture which no longer works. Is there a parsing change and is there a solution to reproduce the layout I have in 1.3.433?

Thanks for your help

#### 1.3.433 Sample
<br>

![image](https://github.com/user-attachments/assets/3365adbf-4b04-4dbf-b926-33fcd4956b5a)

#### 1.5.55 Sample
<br>

![image](https://github.com/user-attachments/assets/b30bbd13-d4e6-4533-9983-9b21a9a6f22d)

### Reproducible Example Code
````qmd
---
title: Missing and Exception
execute:
  echo: false
  message: false
  warning: false
jupyter: python3
---

```{python}
#| label: import packages
from IPython.display import display, Markdown
```

::: {.panel-tabset}

## Missing Values

```{python}
#| label: missing analysis
print('Missing')
over_50_percent_missing = ['admissions']
less_5_percent_missing = ['score', 'letter grade']
display(Markdown(f'''::: {{.callout-caution collapse='true'}}
The following variables have more than 50% missing values: {over_50_percent_missing}.\n
The following variables have less than 5% missing values: {less_5_percent_missing}.
:::'''))
```

## Exception Values
```{python}
print('Exception')
#| label: exception analysis
```
:::
````

**Comment by mcanouil** [comment, marked as answer]
The issue is your labels as it can be seen in your screenshot.
Those are used as ids, so no spaces or non ascii characters.

**Reply by DaedalusInMaze** [reply]
Thanks for helping me pinpointing the issue
